protocol/README.md

```diff
@@ -0,0 +1,55 @@
+# Cardano network Protocol
+
+[![Build Status](https://travis-ci.org/input-output-hk/rust-cardano.svg?branch=master)](https://travis-ci.org/input-output-hk/rust-cardano)
+![MIT or APACHE-2 licensed](https://img.shields.io/badge/licensed-MIT%20or%20APACHE--2-blue.svg)
+![Cardano Mainnet](https://img.shields.io/badge/Cardano%20Ada-mainnet-brightgreen.svg)
+![Cardano Staging](https://img.shields.io/badge/Cardano%20Ada-staging-brightgreen.svg)
+![Cardano Testnet](https://img.shields.io/badge/Cardano%20Ada-testnet-orange.svg)
+
+Support for the [Cardano](https://www.cardano.org) network protocol:
+
+* fetch blocks;
+* send transactions.
+
+## Supported targets
+
+```
+rustup target add aarch64-apple-ios # or any target below
+```
+
+| Target                               | `test` |
+|--------------------------------------|:------:|
+| `aarch64-unknown-linux-gnu`          |   ✓    |
+| `arm-unknown-linux-gnueabi`          |   ✓    |
+| `armv7-unknown-linux-gnueabihf`      |   ✓    |
+| `i686-unknown-linux-gnu`             |   ✓    |
+| `i686-unknown-linux-musl`            |   ✓    |
+| `x86_64-unknown-linux-gnu`           |   ✓    |
+| `x86_64-unknown-linux-musl`          |   ✓    |
+| `i686-apple-darwin`                  |   ✓    |
+| `x86_64-apple-darwin`                |   ✓    |
+| `x86_64-apple-darwin`                |   ✓    |
+| `i686-unknown-freebsd`               |   ✓    |
+| `x86_64-unknown-freebsd`             |   ✓    |
+
+## supported compiler versions
+
+| Rust    | `test` |
+|---------|:------:|
+| stable  |   ✓    |
+| beta    |   ✓    |
+| nightly |   ✓    |
+
+We will always aim to support the current stable. However it is likely
+that older version of the rust compiler is supported too.
```

Comment:
We will always aim to support the current stable version. However, it is likely that an older version of the Rust compiler is also supported.